apps/wasm/mvp/proto/messages.proto

```diff
@@ -0,0 +1,11 @@
+syntax = "proto3";
+
+message PeerStatus
```

Comment:
Doc comments for the interop messages would be sweet

Reply:
Fixed